createdisk.sh

```diff
@@ -340,6 +340,15 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
     echo hv_sock > /etc/modules-load.d/vsock.conf
 EOF
 
+# Add gvisor-tap-vsock service
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
+  podman run -d --name=gvisor-tap-vsock --privileged --net=host -it quay.io/crcont/gvisor-tap-vsock:v3
+  podman generate systemd --restart-policy=no gvisor-tap-vsock > /etc/systemd/system/gvisor-tap-vsock.service
+  systemctl daemon-reload
+  systemctl start gvisor-tap-vsock.service
+  systemctl enable gvisor-tap-vsock.service
```

Comment:
@guillaumerose I think as part of disk image creation you just need to enable it and everytime that this image boot will automatic start this service, or you also need to start this service on the current running instance?

Reply:
Yes you're right. Removing the start line.